loginsystem/newsfeed/images/News Feed.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<!-- saved from url=(0069)file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/splashpage.html -->
+<html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+  
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  <link rel="stylesheet" type="text/css" href="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/CSS/style.css?version=1">
+
+  <title>News Feed</title>
+
+</head>
+<body>
+
+<div class="grid-container">
+  <div class="item1"><img src="file:///Applications/XAMPP/xamppfiles/htdocs/newsfeed/images/newcolorlogo.png" width="200" height="170" alt=""></div>
```

Comment:
at a glance, the indentation looks off. try re-indenting the file.(you can use the sublime plugin for this)